pt/orm/behaviors/translate.rst

```diff
@@ -1,12 +1,476 @@
+    $this->Articles->save($article);
+
+A segunda maneira de usar para salvar entidades em outro idioma é definir o idioma padrão 
+diretamente para a tabela::
+
+    $article->title = 'Mi Primer Artículo';
+
+    // Use locale() anterior à 3.6
+    $this->Articles->setLocale('es');
+    $this->Articles->save($article);
+
+Definir o idioma diretamente na tabela é útil quando você precisa recuperar e salvar entidades 
+para o mesmo idioma ou quando você precisa salvar várias entidades ao mesmo tempo.
+
+.. _saving-multiple-translations:
+
+Salvando Várias Traduções
+=========================
+
+É um requisito comum poder adicionar ou editar várias traduções em qualquer registro do banco de 
+dados ao mesmo tempo. Isso pode ser feito usando o ``TranslateTrait``::
+
+    use Cake\ORM\Behavior\Translate\TranslateTrait;
+    use Cake\ORM\Entity;
+
+    class Article extends Entity
+    {
+        use TranslateTrait;
+    }
+
+Agora, você pode preencher as traduções antes de salvá-las::
+
+    $translations = [
+        'fr' => ['title' => "Un article"],
+        'es' => ['title' => 'Un artículo']
+    ];
+
+    foreach ($translations as $lang => $data) {
+        $article->translation($lang)->set($data, ['guard' => false]);
+    }
+
+    $this->Articles->save($article);
+
+A partir do 3.3.0, o trabalho com várias traduções foi simplificado. Você pode criar controles de 
+formulário para seus campos traduzidos::
+
+    // Em um template.
+    <?= $this->Form->create($article); ?>
+    <fieldset>
+        <legend>French</legend>
+        <?= $this->Form->control('_translations.fr.title'); ?>
+        <?= $this->Form->control('_translations.fr.body'); ?>
+    </fieldset>
+    <fieldset>
+        <legend>Spanish</legend>
+        <?= $this->Form->control('_translations.es.title'); ?>
+        <?= $this->Form->control('_translations.es.body'); ?>
+    </fieldset>
+
+No seu controlador, você pode organizar os dados normalmente::
+
+    $article = $this->Articles->newEntity($this->request->getData());
+    $this->Articles->save($article);
+
+Isso resultará no seu artigo, que todas as traduções em francês e espanhol serão mantidas.
+Você precisará se lembrar de adicionar ``_translations`` aos campos ``$ _accessible`` 
+da sua entidade também.
+
+Validando Entidades Traduzidas
+------------------------------
+
+Ao anexar ``TranslateBehavior`` a um modelo, você pode definir o validador que deve ser 
+usado quando os registros de conversão são criados/modificados pelo comportamento durante 
+``newEntity()`` ou ``patchEntity()``::
+
+    class ArticlesTable extends Table
+    {
+        public function initialize(array $config)
+        {
+            $this->addBehavior('Translate', [
+                'fields' => ['title'],
+                'validator' => 'translated'
+            ]);
+        }
+    }
+
+O texto acima usará o validador criado por ``validationTranslated`` para entidades traduzidas validadas.
+
+.. versionadded:: 3.3.0
```

Comment:
you are writing docs for 4.x , no need to point to any version added for 3.x series

Reply:
All of these still exist.  Can leave it for the translation.

Reply:
Items before translated too?